internal/scaffold/crd.go

```diff
@@ -187,6 +192,7 @@ func runCRDGenerator(rule genall.OutputRule, root string) (err error) {
 }
 
 func newCRDForResource(r *Resource) *apiextv1beta1.CustomResourceDefinition {
+	trueVal := true
 	crd := &apiextv1beta1.CustomResourceDefinition{
```

Comment:
^here I think we need use/import the v1 version as well

Reply:
See https://github.com/operator-framework/operator-sdk/pull/2275#discussion_r351442109